.gitignore

```diff
@@ -15,3 +15,5 @@ prime
 *.snap
 snap/.snapcraft/state
 src/languages/*.qm
+.vscode/
+CMakeLists.txt.user*
```

Comment:
instead of this can you just ignore `*.user` and remove this and `*.pro.user` at the top